performance.lic

```diff
@@ -0,0 +1,100 @@
+# quiet
+custom_require.call %(events common)
+class Performance
+  include DRC
+
+  def initialize
+    @@did_play ||= false
+    @@no_instrument ||= false
+    @@song_list ||= get_data('perform').perform_options
+    Flags.add('ct-song', 'you finish playing')
+  end
+
+  @@instance = Performance.new
+
+  def self.instance
+    return @@instance
+  end
+
+  def self.stop_play
+    return unless @@did_play
+    return if @@no_instrument
+    @@did_play = false
+    DRC.bput('stop play', 'You stop playing your song', 'In the name of', "But you're not performing")
+    Flags['ct-song'] = true
+  end
+
+  def self.play_song?(blocking = false)
+    return true if @researching
```

Comment:
If `Performance.lic` isn't tracking researching, should this line be removed?

If it is removed, does it need to be put in `crossing-training` in each place `play_song?` is called?